.github/workflows/ci.yml

```diff
@@ -125,6 +130,9 @@ jobs:
         # Use | to start a literal so YAML doesn't complain about the '!' character.
         run: |
           ! git grep 'python_version = "PY2"' '*BUILD'
+      - name: 'No comments on licenses rule'
+        run: |
+          ! buildozer '//tensorboard/...:%licenses' remove_comment
```

Comment:
Buildozer exits non-zero when there was nothing to do (the intended case we want here) but it also exits non-zero for errors: https://github.com/bazelbuild/buildtools/blob/e8b2086fd9d7027b97ae8f49744d570fc6c299b4/buildozer/README.md#error-code

So just using `!` to invert the exit status is over-broad here and would hide legitimate errors.  I think instead it might work to just do

```shell
buildozer '//tensorboard/...:%licenses' remove_comment; test $? = 3
```

Worth checking that that does still correctly catch true positives though.

Reply:
`//tensorboard/...` would not catch ones in `//third_party/...` which is a mistake that I made.

Reply:
Updated to check the specific error code.  Tested that it still correctly catches bad cases in https://github.com/tensorflow/tensorboard/pull/4113